cargo-miri/src/phases.rs

```diff
@@ -93,6 +95,17 @@ pub fn phase_cargo_miri(mut args: impl Iterator<Item = String>) {
     let target = get_arg_flag_value("--target");
     let target = target.as_ref().unwrap_or(host);
 
+    // If cleaning the the target directory & sysroot cache,
+    // delete them then exit. There is no reason to setup a new
+    // sysroot in this execution.
+    if let MiriCommand::Clean = subcommand {
+        let metadata = get_cargo_metadata();
+        clean_sysroot();
+        clean_target_dir(&metadata);
+        eprintln!("Cleaning successful.");
```

Comment:
```suggestion
        clean_target_dir(&metadata);
```
I don't think we usually have such success messages for such simple operations. Absence of an error indicates success. :)